enrichment/wikipedia.py

```diff
@@ -87,28 +116,45 @@ def get_wptitle(record):
     try:
         sites = wd_response.json()["entities"][wd_id]["sitelinks"]
     except KeyError:
-        eprint("wikipedia: Data Error for Record:\n\'{record}\'\n\'{wp_record}\'"
-               .format(record=record,wp_record=wd_response.content))
+        eprint("wikipedia: Data Error for Record:")
+        eprint("\'{record}\'\n\'{wp_record}\'"
+               .format(record=record, wp_record=wd_response.content))
```

Comment:
Da das eine Errormeldung ist, würde ich das auch in ein `eprint` packen. Wenn der gesamte Ausdruck von einer Klammer umschlossen ist, reicht es, wenn du die einzelnen Zeilen mit den Anführungszeichen abschließt und ganz normal umbrichst.
``` python
eprint("wikipedia: Data Error for Record:\n"
       "\'{record}\'\n\'{wp_record}\'"
       .format(record=record, wp_record=wd_response.content))
```